CampusTour/ViewControllers/FeaturedViewController.swift

```diff
@@ -67,82 +61,44 @@ class SearchViewController: UIViewController, UITableViewDelegate, UITableViewDa
         self.present(popupViewController, animated: true, completion: nil)
     }
     
-    //Setup Feed portion of ViewController
-    func setBottomView() {
-        addChildViewController(itemFeedViewController)
-        
-        view.insertSubview(itemFeedViewController.view, belowSubview: searchResultsTableView)
-        itemFeedViewController.view.snp.makeConstraints { (make) in
-            make.top.equalTo(filterBar.snp.bottom)
-            make.leading.equalToSuperview()
-            make.trailing.equalToSuperview()
-            make.bottom.equalToSuperview()
-        }
-        
-        itemFeedViewController.didMove(toParentViewController: self)
+    override func viewWillAppear(_ animated: Bool) {
+        searchManager.attachTo(navigationItem: navigationItem)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        searchManager.detachFrom(navigationItem: navigationItem)
     }
     
     //Setup filter & search portion of ViewController
     func setTopNavBar() {
-        //Create search
-        searchBar = UISearchBar()
-        searchBar.placeholder = "Search"
-        searchBar.delegate = self
-        //seems dumb--will have to check on this
-        for s in searchBar.subviews[0].subviews {
-            if s is UITextField {
-                s.layer.borderWidth = 1.0
-                s.layer.borderColor = Colors.shadow.cgColor
-                s.layer.cornerRadius = 10
-            }
-        }
-        
-        navigationItem.titleView = searchBar
-        definesPresentationContext = false
+        searchManager.delgate = self
+        searchManager.allData = testEvents as [Any] + testPlaces as [Any]
         
         arButton = UIBarButtonItem(title: "AR", style: .plain, target: self, action: #selector(openARMode))
         navigationItem.setRightBarButton(arButton, animated: false)
         
```

Comment:
we need the filterbar.delegate for it to communicate the opening of the modal views